libyul/YulStack.cpp

```diff
@@ -293,9 +294,15 @@ YulStack::assembleWithDeployed(std::optional<std::string_view> _deployName)
 			);
 		}
 	}
+	catch (Error const& _error)
+	{
+		reportCodeGenerationError(_error);
```

Comment:
Based on https://github.com/ethereum/solidity/pull/15610/files#r1868734539 you would just do:
```suggestion
		m_errorReporter.reportCodeGenerationError(_error);
```

Reply:
Done.